src/Common/src/Interop/Windows/kernel32/Interop.PerformanceCounterOptions.cs

```diff
@@ -0,0 +1,386 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        internal partial class PerformanceCounterOptions
```

Comment:
This class looks like it contains a lot of constants which aren't used anywhere. Maybe you should clean in up to only contain what is needed. And does this really belong into Kernel32?